src/components/TasksHeader/TasksHeader.tsx

```diff
@@ -13,7 +14,7 @@ const ICONS = {
     badge: BadgeVariant.SUCCESS,
   },
   pending: {
-    icon: <ProgressIcon size={12} />,
+    icon: <AlertCircle size={12} />,
```

Comment:
Preferred the non-spinner icon for the "work to be done" state. Spinner made me think it was stuck loading.